### PR TITLE
Stop tracking py3 deps; rename to "Python 2 Dropping Database"

### PR DESCRIPTION
**Pull request by encukou**

### CODE REVIEW

File: portingdb/templates/index.html [hunk 1]
```diff
@@ -4,7 +4,9 @@
 {% endblock breadcrumbs %}
 
 {% block titlecontent -%}
-    {{ (the_score * 100) | round(1) }}% – {{ super() }}
+    {{ (the_score * 100) | round(1) }}% →
+    {{ (py2_score * 100) | round(1) }}% –
+    {{ super() }}
```
Comment (hroncok) [inline portingdb/templates/index.html]: The new title is a bit comprehensive to grasp.

File: data/statuses.yaml [hunk 2]
```diff
@@ -26,7 +26,7 @@
 - ident: released
   name: Dual support
   abbrev: "\u2714"
-  color: '7EBA54'
+  color: '5EBA84'
```
Comment (hroncok) [inline data/statuses.yaml]: This one feels particularly ugly to my eyes. But I understand that this is not objective feedback.

**Comment by encukou** [comment]
> Why is "update data" the first commit? Does it reflect the new thing?

GitHub orders the commits based on date. That's not the order I intended – I did some rebasing :(
Could you check that locally?

**Comment by hroncok** [comment]
Oh, GitHub... :D

File: portingdb/templates/index.html (hunk 1 above)
Comment (encukou) [reply portingdb/templates/index.html]: So, I tried explaining it on the main page.

File: data/statuses.yaml (hunk 2 above)
Comment (encukou) [reply data/statuses.yaml]: Design changes do that. If it still looks jarring in a week, let's change it?

File: portingdb/templates/index.html [hunk 3]
```diff
@@ -24,42 +64,68 @@ <h1>
         <div class="col-md-12">
             {{ progress_summary(status_summary, total_pkg_count) }}
         </div>
+        <div class="col-md-12">
+            <div class="main-summary">
+            <div class="summary-sentence">
+                <span class="big-percentage"
+                    style="color: {{ statuses['py3-only'].color }};"
+                >
+                    {{ (the_score * 100) | int }}%
```
Comment (hroncok) [inline portingdb/templates/index.html]: round? at least the title now says 44.9 %, but the page says 44 %.

File: portingdb/htmlreport.py [hunk 4]
```diff
@@ -56,19 +74,17 @@ def hello():
 
     by_status = group_by_status(packages.values())
 
-    the_score = sum(len(by_status[s]) for s in DONE_STATUSES) / len(packages)
+    the_score = len(by_status['py3-only']) / len(packages)
+    py2_score = sum(len(by_status[s]) for s in DONE_STATUSES) / len(packages)
```
Comment (hroncok) [inline portingdb/htmlreport.py]: dropped is in DONE_STATUSES...

File: portingdb/templates/index.html [hunk 5]
```diff
@@ -24,42 +64,68 @@ <h1>
...
+                </span>
+                <span class="stuffing">
+                    Python packages in {{ config['name'] }} support
+                </span>
+                <span class="callout">Python 3 only.</span>
+            </div>
+            <div class="summary-sentence">
+                <span class="big-percentage"
+                    style="color: {{ statuses['released'].color }};"
+                >{{ (py2_score * 100) | int }}%</span>
```
Comment (hroncok) [inline portingdb/templates/index.html]: ...this includes dropped packages and hence is not true as in "X% Python packages in Fedora support Python 3."

File: portingdb/templates/index.html (hunk 3 above)
Comment (encukou) [reply portingdb/templates/index.html]: Alright, let me fit that in

File: portingdb/templates/index.html (hunk 5 above)
Comment (encukou) [reply portingdb/templates/index.html]: Good point. The goal changed now; "dropped" is not really "done". I'm keeping `DONE_STATUSES` as they were and adding a new set.

**Comment by encukou** [comment]
Ooh, nice! I just noticed GitHub links to a diff after I force-push.

**Comment by encukou** [comment]
Thanks!

**Pull request merged**